pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -704,6 +742,7 @@ func (r *Builder) host(hostID string) (host *model.Host, err error) {
 	return
 }
 
+//
```

Comment:
i think we are trying to get rid from those. don't we?

Reply:
+1
yeah, 'go fmt' should drop those so need to avoid that